index.html

```diff
@@ -57,14 +58,299 @@ <h3 class="activity">I am a Full Stack Developer</h2>
+          standard
+        </p>
+      </div>
+      <div class="wrapper1">
+        <div class="box2 html">html</div>
+        <div class="box2 bootstrap">bootstrap</div>
+        <div class="box2 ruby">Ruby</div>
+      </div>
+    </div>
+    <div class="card-project-btn">
+      <button class="btn2">See Project</button>
+    </div>
+  </section>
+<!-- Card 6-->
+  <section class="cards-details">
+    <div class="card-container">
+      <div class="title2">
+        <p>Professional Art Printing Data</p>
+      </div>
+      <div class="sup-txt2">
+        <p class="description3">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+      </div>
+      <div class="wrapper1">
+        <div class="box2 html">html</div>
+        <div class="box2 bootstrap">bootstrap</div>
+        <div class="box2 ruby">Ruby</div>
+      </div>
+    </div>
+    <div class="card-project-btn">
+      <button class="btn2">See Project</button>
+    </div>
+  </section>
+  <!-- Cards End Here-->
+
+  <!--Creating A divider-->
+  <div class="empty-divider">
+    <!--Empty Divider-->
+  </div>
+
+  <!-- About Me-->
+  <section class="aboutme">
+    <div class="primary-text2">
+      <h1 class="greetings">About Me</h1>
+    </div>
+    <div class="header3">
+      <p class="description4">
+        I can help you build a product, feature or website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hesitate to contact me.
+      </p>
+    </div>
+    <div class="aboutme-btn">
+      <button class="btn3">Get My Resume</button>
+    </div>
+    <!--Creating A line divider-->
+    <div class="line-divider">
+      <img src="./images/line-divider.png" alt="img3">
+    </div>
+  </section>
+
+  <section class="skill-master">
+    <!--Language-->
+    <div class="sqr-lan">
+      <div class="sqr-language"></div>
+      <div class="base">Language</div>
+    </div>
+    <div class="item js">Javascript</div>
+    <div class="item ruby">Ruby</div>
+    <div class="item html">Html</div>
+    <div class="item css">Css</div>
+    <!--Frameworks-->
+    <div class="sqr-frame">
+      <div class="sqr-framework"></div>
+      <div class="base">Frameworks</div>
+    </div>
+    <div class="item bootstrap">Bootstrap</div>
+    <div class="item ruby-on-rails">Ruby on Rails</div>
+    <div class="item rspec">RSPec</div>
+    <div class="item capyBara">CapyBara</div>
+    <div class="item selenium">Selenium</div>
+    <!--Skills-->
+    <div class="circle-sk">
+      <div class="circle-skills"></div>
+      <div class="base">Frameworks</div>
+    </div>
+    <div class="item codekit">Codekit</div>
+    <div class="item github">Github</div>
+    <div class="item codepen">Codepen</div>
+    <div class="item gitlab">Gitlab</div>
+    <div class="item terminal">Terminal</div>
+  </section>
```

Comment:
- [ ] This category as well should be laid out with ul and li tags and I strongly suggest that you implement this as it improves the semantics of your code base. 👍 
![image](https://user-images.githubusercontent.com/107795045/219816217-c974683f-5036-4337-8d95-7793adb1b876.png)